projects/npm-tools/packages/changelog-generator/src/yarn.js

```diff
@@ -6,7 +6,7 @@
 const child_process = require('child_process');
 const fs = require('fs');
 
-const {error, info} = require('./console');
+const {info, log} = require('./console');
```

Comment:
I don't like this one (I saw it yesterday, but out of context, and didn't provide any feedback because of that). Usually `log` is much less than `error` in trace systems. So, something like: `critical` > `error` > `warning` > `info` > `log`/`debug`.

In this case we rename `error` to `log` and now it may look like `info` > `log` (or at least it is not clear which is higher up in the hierarchy).

So I would:

- leave it as `error` and rename the exception variables
- rename it as `fail`, `abort`, or something like that...

Reply:
Got you! ok, I finally renamed the exception variable 👍